lms/services/lti_grading/_v13.py

```diff
@@ -14,6 +15,9 @@ class LTI13GradingService(LTIGradingService):
 
     LTIA_SCOPES = [
         "https://purl.imsglobal.org/spec/lti-ags/scope/lineitem",
+        # Although this is implied by scope/lineitem according to the spec
+        # Moodle requires this one explicitly for the GET
```

Comment:
> Is there a related issue for this in Moodle's issue tracker?

I'll this slack thread for future reference https://hypothes-is.slack.com/archives/C1MA4E9B9/p1690896090491669

but I don't think creating the issue in Moodle is worth the 5 minute effort.


> What is the release process for features which require additional scopes? 

This case is an odd one because we require additional scopes declared here but not additional permissions, we already had the more broad scope, the bug in Moodle requires explicitly declaring the `read.only` scope but no extra permissions in the LMS install itself.

This is very different than the LMS specific API scopes (more commonly the canvas ones) were requiring a new scope always requires admin intervention and very carefully handling the release.

> Are you just expecting to release the changes and rely on our app to have the necessary permissions or re-authenticate

If we have any cached tokens we'll request a new one. Caching this type of tokens is a very recent (1 / 2 months) feature in any case.


> is there some process to notify admins of the impending changes?

I don't expect any changes needed on the LMS side. I haven't change anything to test this in our LMS instances while adding the new scope.

There's a general uncertainty in some LMS deploys, there might be that one school that has that one setting in their custom install that makes this particular change a problem. We had has cases like that in the past but I don't think this will be one of those.